package.json

```diff
@@ -5,10 +5,13 @@
   "private": true,
   "license": "MPLv2",
   "scripts": {
+    "start:a": "npm run build && web-ext run -s dist/extension-a-$npm_package_version",
```

Comment:
Instead of `npm run build`, you should use `npm-run-all build`, which will correctly use Yarn if the function was called with Yarn.